ldelf/ta_elf_rel.c

```diff
@@ -327,6 +327,9 @@ static void e32_relocate(struct ta_elf *elf, unsigned int rel_sidx)
 		size_t sym_idx = 0;
 		vaddr_t val = 0;
 
+		if (ELF32_R_TYPE(rel->r_info) == R_ARM_NONE)
+			continue;
```

Comment:
Maybe I was a bit rude testing this here. I cannot find back where I saw the offset value was not to consider, at least for 32b and 64b Arm architecture.

Maybe a better fix would be to move this test in the switch/case below:
```patch
 		switch (ELF32_R_TYPE(rel->r_info)) {
+		case R_ARM_NONE:
+			break;
 		case R_ARM_ABS32:
```

Reply:
ok, i'll fix.